tests/schemas/rules/boolean/value/count.json

```diff
@@ -0,0 +1,98 @@
+[
+  {
+    "description": "A checkbox selected answer equal to number of checkboxes count number",
```

Comment:
This needs to be clearly as I don't really understand what a checkboxes count number is.

Reply:
From the existing file, but have updated to be meaningful.